supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/CpuDeclarationValidation.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.backend.impl.sfs.config
+
+import cromwell.backend.validation.{RuntimeAttributesValidation, ValidatedRuntimeAttributes}
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Positive
+import wdl.draft2.model.Declaration
```

Comment:
ToL: It obviously compiles so I guess so, but is this really a `WDL` `Declaration`?

Reply:
Yep the config backend uses draft 2 wdl